src/psyclone/tests/psyir/nodes/assignment_test.py

```diff
@@ -314,3 +315,18 @@ def test_is_not_array_range():
     assignment = Assignment.create(StructureReference.create(grid_sym, ["dx"]),
                                    one.copy())
     assert assignment.is_array_range is False
+
+
+def test_assignment_gen_code():
+    '''Test that the create method in the Assignment class correctly
```

Comment:
Previous tests do this. Presumably this is really testing the `gen_code` method and is only needed to get coverage of otherwise dead code that you talked about earlier? If so, please could you add a TODO and refer to the appropriate issue.

Reply:
I updated the docstring and added a reference to a new issue.

Reply:
Thanks.